packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1322,16 +1727,22 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
         controller: _primaryScrollController,
         child: new Material(
           color: widget.backgroundColor ?? themeData.scaffoldBackgroundColor,
-          child: new CustomMultiChildLayout(
-            children: children,
-            delegate: new _ScaffoldLayout(
-              statusBarHeight: mediaQuery.padding.top,
-              bottomViewInset: widget.resizeToAvoidBottomPadding ? mediaQuery.viewInsets.bottom : 0.0,
-              endPadding: endPadding,
-              textDirection: textDirection,
-              geometryNotifier: _geometryNotifier,
-            ),
-          ),
+          child: new AnimatedBuilder(animation: _floatingActionButtonMoveController, builder: (BuildContext context, _) {
```

Comment:
usually we prefer to type and name all the arguments, even those we don't use, so that people reading the code can tell at a glance what they are (rather than having to go look it up in the docs)

Reply:
Done